setup.cfg

```diff
@@ -1,7 +1,7 @@
 [metadata]
 # replace with your username:
 name = luna-fviz
-version = 0.2
+version = 0.3
```

Comment:
Again, let's do this once we want to re-release. Probably once all this has been applied.

Reply:
Reverted to 0.2
I will just change the version in the last branch to be merged.